editor/editor_node.cpp

```diff
@@ -7257,6 +7273,28 @@ EditorNode::EditorNode() {
 	load_error_dialog->set_title(TTR("Load Errors"));
 	gui_base->add_child(load_error_dialog);
 
+	long_load_dialog = memnew(ConfirmationDialog);
+	{
+		VBoxContainer *vbc = memnew(VBoxContainer);
+		long_load_dialog->add_child(vbc);
+
+		Label *dl = memnew(Label);
+		dl->set_text(TTR("Your project seems to have been loading for a long time due to previously opened scenes. Do you want to disable scene restoring?"));
```

Comment:
I'd change the wording to make it clear that the setting only affects future sessions of the editor:

```suggestion
		dl->set_text(TTR("Your project seems to have been loading for a long time due to previously opened scenes. Do you want to disable scene restoring in future editor sessions?"));
```